snap/snapcraft.yaml.in

```diff
@@ -298,9 +298,11 @@ parts:
                   -X '${version_package}.OvsVersion=${ovs_pkg_version}'"
```

Comment:
Now that autoconf is the authoritative source of the Git part of the version string, should we not use `@VERSION@` instead of `${git_version}` in `snapcraft.yaml.in`?

Reply:
Oh! that is a good point, I didn't think of that. Yes, we should do that.